### DISCUSSION TITLE
After Quick Start,there is an error:ConnectionError

### DISCUSSION THREAD

**Discussion by JianJianMa** [Skyvern-AI/skyvern#537] [Q&A]
: HTTPConnectionPool(host='127.0.0.1', port=8000): Max retries exceeded with url: /api/v1/internal/tasks?page=1&page_size=15 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff416a9890>: Failed to establish a new connection: [Errno 111] Connection refused'))
how can i resolve this problem?thanks

**Comment by suchintan** [comment]
Did you do the docker quick start? Or setup.sh

**Reply by JianJianMa** [reply]
I start it using 'docker compose up -d',and navigate to http://localhost:8501/,then error happened.

**Reply by suchintan** [reply]
Interesting! I think @msalihaltun and @wintonzheng rolled out a fix for this recently. Have you tried doing `docker compose pull` and then `docker compose up -d` again?
